src/pages/token/rewards/RewardHistoryPanel.tsx

```diff
@@ -57,7 +58,9 @@ export const RewardHistoryPanel = () => {
                   REWARDS_HISTORY_START_DATE: (
                     <$Output
                       type={OutputType.Date}
-                      value={REWARDS_HISTORY_START_DATE_MS}
+                      value={parseInt(
+                        ENVIRONMENT_CONFIG_MAP[selectedNetwork].rewardsHistoryStartDateMs
```

Comment:
I feel like there should be a helper hook/function that just takes in the key you want (rewardsHistoryStartDateMs) and returns the value - doing the env lookup on its own. Like components shouldn't need to know about environments and selected network and things.

Reply:
ah fair ya - I took a brief peek around the codebase but didn't see one to do so already so just went the lazy route ahaaaa. Can make one and update